Add extraEnv & remove duplicate label

covert8 opened the pull request:
* Enables adding listen env var for ipv6 support fixing #2 

* Corrects duplicate label "helm.sh/charts" already present in "oci-registry.labels" causing kustomize builds to fail

covert8 committed:
Add extraEnv to deployment.yaml

Remove trailing line
Add trailing space to brackets

mcronce commented:
@covert8 thanks for the contribution!  This looks good to me - the only request I have is to undo the indentation changes in `deployment.yaml`, then I'll merge and cut a release

covert8 committed:
fixup! Add extraEnv to deployment.yaml

covert8 committed:
Remove duplicate labels causing error in kustomize build

$ kustomize build . --enable-helm

Error: map[string]interface {}(nil): yaml: unmarshal errors:
  line 11: mapping key "helm.sh/chart" already defined at line 10

covert8 committed:
Add extraEnv to values.yaml

covert8 commented:
Thanks for working with me, you'll have to squash the pr but i hope its ok now?

mcronce commented on dist/helm/templates/deployment.yaml:
```diff
@@ -1,11 +1,12 @@
+  name:  {{ template "oci-registry.fullname" . }}
   labels:
     {{- include "oci-registry.labels" . | nindent 4 }}
+    helm.sh/chart: {{ template "oci-registry.chart" . }}
```
You undid one of your changes :)

No problem though - when I get done my dayjob today I'll merge, fix this, and cut a release

Thanks for fixing it up!

covert8 replied:
thanks for noticing, fixed now

mcronce commented:
@covert8 this fix is included in the now-released v0.3.11.  Thanks again!
